test/create-request-client.js

```diff
@@ -471,4 +471,47 @@ describe('request client', () => {
       body.url.should.eql('http://zapier-httpbin.herokuapp.com/get');
     });
   });
+
+  describe('shorthand hook subscriptions', () => {
+    it('should resolve bundle tokens in performSubscribe', () => {
+      const targetUrl = 'https://zapier.com/hooks';
+      const event = {
+        bundle: { targetUrl }
+      };
+      const subscribeInput = createInput({}, event, testLogger);
+      const request = createAppRequestClient(subscribeInput);
+      return request({
+        url: 'http://zapier-httpbin.herokuapp.com/post',
+        method: 'POST',
+        body: {
+          hookUrl: '{{bundle.targetUrl}}'
+        }
+      }).then(response => {
+        const { hookUrl } = JSON.parse(response.request.body);
```

Comment:
Is it intended? This doesn't check the echoed content from httpbin. Should probably change to:

```suggestion
        const hookUrl = response.json.json.hookUrl;
```

Reply:
I see 👍. I was copying a similar pattern for validating the body from another test in the suite. Probably a lack of knowledge about what httpbin sends.

Would checking body on the request suffice though? We just want to know that we resolved `{{bundle.targetUrl}}` to the value of `hookUrl`

Reply:
Since we can be almost 100% certain that httpbin always echoes back the content we sent to it, checking the response from httpbin is probably more reliable than checking `response.request.body`, which is done by us in `core`. Does that make sense?

Reply:
That does. Thanks for explaining @eliangcs!